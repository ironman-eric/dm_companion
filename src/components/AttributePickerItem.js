function AttributePickerItem({ name, children, onOptionChange }) {
  return (
    <span className="inline-block bg-gray-200 rounded-full px-3 py-1 text-sm font-semibold text-gray-700 mr-2">
      <input type="radio" name={name} value={children} onChange={onOptionChange} /> &nbsp;
      {children}
    </span>
  );
}

export default AttributePickerItem;
