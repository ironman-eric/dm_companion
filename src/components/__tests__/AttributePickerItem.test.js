import React from "react";
import renderer from "react-test-renderer";
import AttributePickerItem from "../AttributePickerItem";

describe("AttributePickerItem", () => {
  it("Matches DOM Snapshot", () => {
    const domTree = renderer.create(<AttributePickerItem />).toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
