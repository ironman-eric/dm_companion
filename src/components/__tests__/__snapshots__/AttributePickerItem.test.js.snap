// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AttributePickerItem Matches DOM Snapshot 1`] = `
<span
  className="inline-block bg-gray-200 rounded-full px-3 py-1 text-sm font-semibold text-gray-700 mr-2"
>
  <input
    type="radio"
  />
   Â 
</span>
`;
